lib/messages/local/local_ack.hpp

```diff
@@ -3,27 +3,26 @@
 
 #include <cstdint>
 #include <messages/local/local_spa_message.hpp>
+#include <messages/op_codes.hpp>
 
 struct LocalAck : public LocalSpaMessage
 {
   LocalAck(
-    uint8_t version,
-    uint8_t priority,
-    LogicalAddress destination,
-    LogicalAddress source,
-    uint16_t flags,
-    uint16_t sourcePort,
-    uint8_t status 
-  ):LocalSpaMessage(
-    version,
-    priority,
-    72,
-    destination,
-    source,
-    flags,
-    0x21,
-    sourcePort
-  ), status(status){}
+      uint8_t version,
```

Comment:
I also like the formatting changes, but in the future if something needs to be formatted, branch again from master, format, get it merged, and then pull it into your branch. Not a huge deal now, but it is a good practice